src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsExtraMarkerOasis.java

```diff
@@ -0,0 +1,42 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+/**
+ * @ownership AdEngineering
+ */
+public class TestAdsExtraMarkerOasis extends TemplateNoFirstLoad {
+
+  @Test(
+      dataProviderClass = AdsDataProvider.class,
+      groups = "AdsExtraMarkerOasis",
+      dataProvider = "adsExtraMarkerOasis"
+  )
+  public void AdsExtraMarkerOasis(final String wikiName,
+                                  final String article,
+                                  final String adType,
+                                  final String message) {
+    String testedPage = urlBuilder.getUrlForPath(wikiName, article);
+    final AdsBaseObject wikiPage = new AdsBaseObject(driver, testedPage);
+    new WebDriverWait(driver, 30).until(new ExpectedCondition<Boolean>() {
```

Comment:
Magic number! Is it 30 seconds? Can we put it in a const with a descriptive name?